pandas/core/indexing.py

```diff
@@ -1595,6 +1595,15 @@ def _get_slice_axis(self, slice_obj, axis=0):
         else:
             return self.obj.take(slice_obj, axis=axis, convert=False)
 
+    def _get_list_axis(self, key_list, axis=0):
+
```

Comment:
Could you give a hint on phrasing please?
Would "return Series values by list or array of integers" be clear enough?